src/libraries/Common/src/Roslyn/SyntaxValueProvider_ForAttributeWithSimpleName.cs

```diff
@@ -0,0 +1,295 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+
+using Microsoft.CodeAnalysis;
+
+using Aliases = System.Collections.Generic.ValueListBuilder<(string aliasName, string symbolName)>;
+
+namespace Microsoft.CodeAnalysis.DotnetRuntime.Extensions;
+
+internal static partial class SyntaxValueProviderExtensions
+{
+    // private static readonly ObjectPool<Stack<string>> s_stackPool = new(static () => new());
```

Comment:
runtime doesn't pool.  so got rid of this.  we are using a ValueListBuilder instead, so it should still be efficient.